### ISSUE TITLE
one of the variables needed for gradient computation has been modified by an inplace operation

### ISSUE THREAD

**Issue by ladybirdhui** [Tshzzz/pytorch_yolov2#2]
I think it is because i don't use .detach() somewhere but i don't know where it is .......

**Comment by Tshzzz** [comment]
I'm sorry... You should remove loss_iou in the loss funtion, line 143!

**Comment by Tshzzz** [comment]
I'll fix it right away....

**Comment by Tshzzz** [comment]
You can see the repository https://github.com/Tshzzz/jinnan_yolo_baseline . This repository had been tested today.

**Comment by ladybirdhui** [comment]
thank you 
i will try
